<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>German</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>French</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arabic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chinese</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Belorussian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hebrew</string>
    </property>
   </item>
  </widget>
  <widget class="QProgressBar" name="accuracyProgressBar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>symbols in minute</string>
   </property>
  </widget>
  <widget class="QLabel" name="valueLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
